Documentation/Helm-Charts/operator-chart.md

```diff
@@ -46,8 +46,28 @@ The following table lists the configurable parameters of the rook-operator chart
 |-----------|-------------|---------|
 | `allowLoopDevices` | If true, loop devices are allowed to be used for osds in test clusters | `false` |
 | `annotations` | Pod annotations | `{}` |
+| `ceph-csi-operator.enabled` |  | `false` |
+| `ceph-csi-operator.fullnameOverride` |  | `"ceph-csi-operator"` |
+| `ceph-csi-operator.nameOverride` |  | `"ceph-csi-operator"` |
 | `cephCommandsTimeoutSeconds` | The timeout for ceph commands in seconds | `"15"` |
+| `cephfsCtrlpluginSa.serviceAccount.annotations` |  | `{}` |
+| `cephfsNodepluginSa.serviceAccount.annotations` |  | `{}` |
 | `containerSecurityContext` | Set the container security context for the operator | `{"capabilities":{"drop":["ALL"]},"runAsGroup":2016,"runAsNonRoot":true,"runAsUser":2016}` |
+| `controllerManager.manager.args[0]` |  | `"--leader-elect"` |
```

Comment:
Is there any way we can have these settings be under csi, for example with a `csi-operator` prefix? Currently, they are just spread all over the doc, and can't tell what belongs to rook or csi.

Reply:
I can move setting to under csi. I'll do the change